docs/dev-setup.md

```diff
@@ -56,5 +56,5 @@ Please follow metal3 dev guide for setting up above prerequisites -
    make run
    ```
 
-Note: Setup is completed here. To use HWCC follow HWCC user-guide
-(Link will be shared soon)
+Note: Setup is completed here. To use HWCC follow HWCC
```

Comment:
Only keep to use HWCC, follow [User guide](user-guide.md)

Anyway you included link to guide so don't need mention specifically HWCC again.

Reply:
done.